packages/components/tree-v2/src/virtual-tree.ts

```diff
@@ -87,6 +87,10 @@ export const treeProps = buildProps({
     type: Number,
     default: 16,
   },
+  itemSize: {
```

Comment:
**Question:** why were we adding this twice?

Reply:
Oh, my mistake, they were added to two different props, perhaps we can do something like this?

```ts
const itemSize = {
  type: Number,
  default: 26,
}

const treeNodeProps = {
  ...others,
  itemSize,
}
```

Reply:
Looks good.